airflow/providers/openlineage/extractors/manager.py

```diff
@@ -37,13 +37,13 @@ def __init__(self):
 
         # Comma-separated extractors in OPENLINEAGE_EXTRACTORS variable.
         # Extractors should implement BaseExtractor
-        from airflow.providers.openlineage.utils import import_from_string
```

Comment:
(Can be separate PR) Is that `import_from_string` used anywhere? If not should we remove it/remove the uses of it?

Reply:
Actually, that was a broken import that was never used in #29940 and not detected as a bug. That was the only one use.